tailwind.config.js

```diff
@@ -16,6 +16,53 @@ module.exports = {
     },
     fontFamily: {
       body: ["Outfit"]
+    },
+    height: {
+      "cg-s":"50rem",
+      "cg-l":"64.208rem"
+    },
+    maxWidth:{
+      "cg-l":"94.625rem",
+      "cg-s":"6.088rem",
+      "cg-l-herodiv1":"14.75rem",
+      "cg-s-herodiv2":"18.727rem",
+      "cg-l-herodiv2":"45.375rem",
```

Comment:
Using a camel case naming convention to name variables is a good practice. It increases the readability and maintainability of the codebase. Example `primaryBlue`, `sumOfNumbers`